packages/adyen-integration/src/adyenv3/adyenv3-payment-strategy.spec.ts

```diff
@@ -184,8 +187,8 @@ describe('AdyenV3PaymentStrategy', () => {
                 );
             });
 
-            it('throws an error when payment is not present', () => {
-                expect(strategy.execute(getOrderRequestBodyWithoutPayment())).rejects.toThrow(
+            it('throws an error when payment is not present', async () => {
+                await expect(strategy.execute(getOrderRequestBodyWithoutPayment())).rejects.toThrow(
```

Comment:
why do we need this change?

Reply:
Actually after analyzing it, it doesn’t affect tests at all.
I was getting the log _„Cannot log after tests are done. Did you forget to wait for something async in your test?”_ when I was running tests locally. I was trying to fix it with adding async/await (in execute method, there is asynchronous code and all execute calls in test file, except this one uses async/await) but it didn’t help. In CI environment this log doesn’t show up.
I removed this change.